plugins/controlboard/src/ControlBoard.cc

```diff
@@ -23,7 +23,8 @@ namespace gazebo
 
 GZ_REGISTER_MODEL_PLUGIN(GazeboYarpControlBoard)
 
-    GazeboYarpControlBoard::GazeboYarpControlBoard() : m_iWrap(0)
+    GazeboYarpControlBoard::GazeboYarpControlBoard() : m_iWrap(0),
+                                                       m_iVirtAnalogSensorWrap(0)
```

Comment:
should we change this to `0` to `nullptr` ? here and elsewhere?

Reply:
good point:
```suggestion
                                                       m_iVirtAnalogSensorWrap(nullptr)
```